elasticdl/python/master/task_manager.py

```diff
@@ -293,6 +308,11 @@ def create_tasks(self, task_type, model_version=-1):
                     start_idx_this_task + self._records_per_task,
                     max_idx_this_shard,
                 )
+                task_record_indices = task_record_indices = (
```

Comment:
`task_record_indices = task_record_indices = xxx`.
Is it a typo?

Reply:
Yes, fix it!